src/validators/format.js

```diff
@@ -0,0 +1,20 @@
+const checker = require("../checker");
+const err = require("../errorCodes");
+
+function format(value, expression) {
+  let results = null
+  if (checker.isEmpty(value)) {
```

Comment:
I'm trying to keep simple guard clauses [1] like that as a single line for readability reasons 
[1] https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html